applications/challengedac.md

```diff
@@ -0,0 +1,109 @@
+
+# EOS Network Foundation Grant Proposal
+
+
+- **Project Name:** ChallengeDac
+- **Team Name:** ChallengeDac
+- **EOS Payment Address:** g4ztaojzgage
+- **[Level](https://github.com/eosnetworkfoundation/grant-framework#grant-levels):** 2
+- **Pomelo Grant(s):** [https://pomelo.io/grants/chlcontent](https://pomelo.io/grants/chlcontent)
+- **Project is Open-Source:** Yes
+- **Project was part of Token sale:** No
+- **Repository where Project resides:** [https://github.com/ChallengeDAC](https://github.com/ChallengeDAC)
+
+## Contact
+
+- **Contact Name:** Chase Eskam
+- **Contact Email:** [chaceaeskam@gmail.com](mailto:chaceaeskam@gmail.com)
+- **Website:** [https://www.challengedapp.io/](https://www.challengedapp.io/)
+
+
+## Project Overview
+
+
+
+### Overview
+
+ChallengeDac is a decentralized autonomous organization (DAO) built on the EOS and Telos blockchain that aims to provide a platform for people to create, discover, and participate in Challenges. ChallengeDAC allows users to create Challenges in a variety of categories, such as education, fitness, and entertainment, and to set the terms and conditions for participation. Other users can then discover and participate in these challenges, earning rewards for successfully completing them. ChallengeDAC uses smart contracts to enforce the rules of each challenge and to automate the distribution of rewards. CurrentlychallengeDac has a user's base of over 20,000.
```

Comment:
What is the current state of application? Are these 20k active users of this app? Looks like open repos on github has only flutter scaffold of the frontend of the app. I may be looking in the wrong place?

Reply:
Also wondering if play store and iTunes would have no issues with app with crypto award for app promotion / downloads / service / etc in their store.

Reply:
Can you please reaffirm that based on TOS for iTunes and Play store that pay-per-challenge app would be allowed.

Reply:
Yes we have close to a 3 year history of being approved and available by IOS and google play.